extensions-contrib/statsd-emitter/src/main/java/org/apache/druid/emitter/statsd/StatsDEmitter.java

```diff
@@ -174,6 +177,59 @@ public void emit(Event event)
       } else {
         log.debug("Service=[%s], Metric=[%s] has no StatsD type mapping", service, metric);
       }
+    } else if (event instanceof AlertEvent && config.isDogstatsd() && config.isDogstatsdEvents()) {
+      AlertEvent alertEvent = (AlertEvent) event;
+
+      ImmutableMap.Builder<String, String> tagBuilder = ImmutableMap.builder();
+
+      tagBuilder
+          .put("feed", alertEvent.getFeed())
+          .put("service", alertEvent.getService())
+          .put("severity", alertEvent.getSeverity().toString());
+      if (config.getIncludeHost()) {
+        tagBuilder.put(TAG_HOSTNAME, alertEvent.getHost());
+      }
+
+      String text;
+      try {
+        text = mapper.writeValueAsString(alertEvent.getDataMap());
+      }
+      catch (JsonProcessingException e) {
+        log.error(e, "Unable to convert alert data to json");
+        text = "Unable to convert alert data to JSON: " + e.getMessage();
+      }
+      statsd.recordEvent(
+          com.timgroup.statsd.Event
+              .builder()
+              .withDate(alertEvent.getCreatedTime().getMillis())
+              .withAlertType(alertType(alertEvent.getSeverity()))
+              .withPriority(com.timgroup.statsd.Event.Priority.NORMAL)
+              .withTitle(alertEvent.getDescription())
+              .withText(text)
+              .build(),
+          tagsFromMap(tagBuilder.build())
+      );
+    }
+  }
```

Comment:
We can probably separate the two sections for better readability/maintainability. Example:

```
    if (event instanceof ServiceMetricEvent) {
        submitMetrics()
    } else if (event instanceof AlertEvent && config.isDogstatsd() && config.isDogstatsdEvents()) {
        submitEvents()
    }
```

Reply:
done